src/containers/SortingPanel.container.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import SortingPanel from '../components/SortingPanel'
+
+const mapStateToProps = state => ({
+    state
```

Comment:
as I can see from SortingPanel code, it really need only sortingType from the state, so please pass only this argument instead of passing whole state

Reply:
done